src/components/ContributorShowcase/ContributorShowcase.module.css

```
@@ -0,0 +1,14 @@
+.grid {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+}
+
+.image {
+  width: 100px;
+  height: 100px;
+}
+
+.griditem {
+  text-align: left;
```

We can use the [Tailwind syntax](https://tailwindcss.com/docs/text-align) here.
I think it looks nicer with center aligned text. Also, seemed a little big to me, so made the text smaller.
Hope that's okay.

```suggestion
  @apply text-xs text-center;
```

---

perfect, thanks for everything